src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -468,7 +468,7 @@
     text: az aks create -g MyResourceGroup -n MyManagedCluster --kubernetes-version 1.16.9
   - name: Create a Kubernetes cluster with a larger node pool.
     text: az aks create -g MyResourceGroup -n MyManagedCluster --node-count 7
-  - name: Create a kubernetes cluster with k8s 1.13.9 but use vmas.
+  - name: Create a kubernetes cluster with k8s 1.13.9 but use vmss.
```

Comment:
Maybe this means Virtual Machine Availability Set? Help messages should avoid using aberrations to accuratly convey concepts.

Reply:
@maulikramani07  Any update?

Reply:
As shown in the example in the next line, here is indeed an attempt to create a cluster with`vmas` (**V**irtual**M**achine**A**vailability**S**et), so I suppose this is not a typo?

Reply:
@maulikramani07 Could you please take a look at this comment?